vignettes/mapmetadata.Rmd

```diff
@@ -31,13 +31,13 @@ It is important to note that this plot is only summarising *variable* level meta
 
 The bar plot can help you understand the scope of the dataset, but reference the HDRUK Gateway page for the fuller context. For instance, table descriptions are not included in these structural metadata files but they are included on the gateway:
 
-**For dataset NCCHD, used in the demo, the structural metadata was downloaded here:**
+For dataset NCCHD, used in the demo, the structural metadata was downloaded from https://healthdatagateway.org/en/dataset/360. Below, we include a screenshot of this page; a user must click on the 'Download Data' then 'Structural Metadata' in order to download the file. Please note, we are working on API intergration with this package to streamline this for users.
```

Comment:
Small error - should be integration. This also affects an html file not in the PR BAR_360_NationalCommunityChildHealthDatabase(NCCHD)_2024-12-19-14-11-55.html

Reply:
Oh dear, I should get better at using spell checkers. 
Thank you! 
I have correct it now.

Reply:
When you say 
`This also affects an html file not in the PR BAR_360_NationalCommunityChildHealthDatabase(NCCHD)_2024-12-19-14-11-55.html`
Just trying to work out what you mean. Can you expand? 
Many thanks

Reply:
I did a global search to look for the word and found it also appeared in the html in `inst/outputs/BAR_360_.....html` and just appears in the comments so won't show up unless searched for. If you wanted to change it I'd probably just do it directly in the file.

Reply:
On the spelling, I'm ok with that but terrible with grammar so it's often a team effort for documentation 😄

Reply:
Oh yes, good spot! I see what you mean now 
That is someone else's spelling mistake
>     / AFAIK, leaflet is the only library with such intergration
As it is in the raw html (and I did not generate it) I will leave it for now